src/stylesheets/components/_tabs.scss

```diff
@@ -152,27 +152,9 @@
   }
 }
 
-.app-tabs__container pre {
-  max-width: inherit;
-  margin-bottom: 0;
-  padding: 0;
-  border: $govuk-focus-width solid transparent;
+.app-tabs__container pre code {
```

Comment:
Newer `highlight.js` updates switched from scrolling `<pre>` to `<code>` which prepares us for:

* https://github.com/alphagov/govuk-design-system/pull/2561